linkml-schema/reproschema.yaml

```diff
@@ -482,9 +457,15 @@ classes:
     class_uri: reproschema:LandingPage
   langString:
     description: RDF langString tuple
-    slots:
-    - langstring_prefix
-    - langstring_value
+    attributes:
+      en:
+        title: English
+        description: English language string
+        range: string
+      es:
+        title: Spanish
+        description: Spanish language string
+        range: string
```

Comment:
see my comment on different language notations. we may not be able to do this for all language specifications. however, we could add a validator. in pydantic it's possible to inject one later to map into an enumeration. let's keep the dictionary form for now.

Reply:
I have added `extra=Extra.allow` so you don't have to list all.

Reply:
you mean you want to have just `Dict[str, str]`?

Reply:
did you see the keys in the section i linked to some of them are of the form `pt-BR` and `pt-PL` any hyphenated key cannot be represented as an attribute.

Reply:
yes, `Dict[Str, Str]` for now to handle these hyphenated language forms.

Reply:
> did you see the keys in the section i linked to some of them are of the form `pt-BR` and `pt-PL` any hyphenated key cannot be represented as an attribute.

not sure what you referring to

Reply:
@satra - but I will also change the places in python where you have `langString` (e.g. in `value` field) to `Dict[str, str]` (and we can actually remove `langString` fro  pydantic, since nothing would be able to pass the validation with the current keys)